backend/app/tests/api/routes/test_users.py

```diff
@@ -372,42 +372,53 @@ def test_update_user_email_exists(
     assert r.json()["detail"] == "User with this email already exists"
 
 
-def test_delete_user_super_user(
-    client: TestClient, superuser_token_headers: dict[str, str], db: Session
-) -> None:
+def test_delete_user_me(client: TestClient, db: Session) -> None:
     username = random_email()
     password = random_lower_string()
     user_in = UserCreate(email=username, password=password)
-    user = crud.create_user(session=db, user_create=user_in)
-    user_id = user.id
+    _ = crud.create_user(session=db, user_create=user_in)
```

Comment:
```suggestion
    user = crud.create_user(session=db, user_create=user_in)
    user_id = user.id
```

this way we can get the `user_id` generated by the DB and later use it to confirm that it was deleted in the DB.